src/applications/combined-debt-portal/combined/components/Balances.jsx

```diff
@@ -0,0 +1,153 @@
+import React from 'react';
+import head from 'lodash/head';
+import moment from 'moment';
+import { useSelector } from 'react-redux';
+import BalanceCard from './BalanceCard';
+import ZeroBalanceCard from './ZeroBalanceCard';
+import AlertCard from './AlertCard';
+import ComboAlerts from './ComboAlerts';
+
+export const IS_DEBT = true;
+
+export const calculateTotalDebts = debts => {
```

Comment:
CODE STRUCTURE: These exported functions should be in their own file, like a `utils` or something more descriptive and less involved in the jsx. JSX files should be dealing with displaying only and the logic should be someplace else, for re-use and isolation.